src/components/molecules/Visualizer/Engine/Cesium/cameraLimiter.ts

```diff
@@ -0,0 +1,237 @@
+import {
+  Cartesian3,
+  Cartographic,
+  Color,
+  EllipsoidGeodesic,
+  PolylineDashMaterialProperty,
+  Math,
+  Camera as CesiumCamera,
+  Rectangle,
+} from "cesium";
+import type { Viewer as CesiumViewer } from "cesium";
+import { useEffect, useMemo, useState } from "react";
+import { CesiumComponentRef } from "resium";
+import { RefObject } from "use-callback-ref/dist/es5/types";
+
+import { Camera } from "@reearth/util/value";
+
+import type { SceneProperty } from "..";
+
+import { getCamera } from "./common";
+
+export function useCameraLimiter(
+  cesium: RefObject<CesiumComponentRef<CesiumViewer>>,
+  camera: Camera | undefined,
+  property: SceneProperty["cameraLimiter"] | undefined,
+) {
+  const geodsic = useMemo(():
```

Comment:
```suggestion
  const geodesic = useMemo(():
```
!! Don't forget to update wherever this variable is being used. !!